src/utils/string/index.ts

```diff
@@ -93,3 +95,16 @@ export const getBytesAsString = (bytes: number, decimals = 2) => {
   const i = Math.floor(Math.log(bytes) / Math.log(k));
   return `${parseFloat((bytes / k ** i).toFixed(dm))} ${sizes[i]}`;
 };
+
+/**
+ * `trimLineToMaxSize` trims a line to the max size limit
+ * @param line - the line to trim
+ * @returns the trimmed line
+ * @example
+ */
+export const trimLineToMaxSize = (line: string) => {
+  if (line.length > LOG_LINE_SIZE_LIMIT) {
+    return `${line.substring(0, LOG_LINE_SIZE_LIMIT)}...`;
```

Comment:
[nit] Could use a single ellipsis character here instead of 3 periods
```suggestion
    return `${line.substring(0, LOG_LINE_SIZE_LIMIT)}…`;
```